install.sh

```diff
@@ -406,23 +376,20 @@ main() {
 
   adjust_os
 
-  if [ ${OS} = "windows" ]; then
-    FORMAT=zip ;
-  fi
-
-  log_info "found version: ${VERSION} for ${TAG}/${OS}/${ARCH}"
-
   # If < v3, archive version is prefixed with "v"
+  VERSION=${TAG}
```

Comment:
This doesn't seem to work if the tag is `latest` because then it tries to download `TARBALL=${BINARY}_latest_${OS}_${ARCH}.${FORMAT}`